src/view/actions/sendEvent.jsx

```diff
@@ -125,26 +83,21 @@ const getSettings = ({ values }) => {
   if (values.renderDecisions) {
     settings.renderDecisions = true;
   }
-  const scopes = getDecisionScopesFromFormState(values);
-  if (scopes) {
-    settings.decisionScopes = scopes;
-  }
+
+  Object.assign(settings, getDecisionScopesSettings({ values }));
 
   return settings;
 };
 
 const validationSchema = object().shape({
   xdm: string().matches(singleDataElementRegex, {
-    excludeEmptyString: true,
     message: "Please specify a data element"
   }),
```

Comment:
Now that you removed the excludeEmptyString key, you can just pass the message as the second parameter.